src/mixins/filing-template-mixin.ts

```diff
@@ -63,6 +64,8 @@ export default class FilingTemplateMixin extends Vue {
   @Action setStaffPayment!: ActionBindingIF
   @Action setDetailComment!: ActionBindingIF
   @Action setOriginalSnapshot!: ActionBindingIF
+  @Action setProvisionsRemoved!: ActionBindingIF
+  @Action setHasProvisions!: ActionBindingIF
```

Comment:
This is not used; please delete.

Also please clean up any actions/mutations/state if "hasProvisions" is not used at all.

Reply:
Yes, that's right. Thanks!

Reply:
I think you set this unused state in the tests too.

Reply:
No, I checked again. We got rid of it!